app/build.gradle

```diff
@@ -109,6 +112,16 @@ android {
     }
 }
 
+spotless {
+    java {
+        removeUnusedImports()
+        indentWithSpaces(2)
```

Comment:
Why 2 spaces?

Reply:
it was not matching the android studio's formatting standards with default 1 space.

eg: 
1 Space:
```
public class AboutUsFragment extends BaseFragment {
  private AboutUsFragmentBinding binding;
```
2 Space:
```
public class AboutUsFragment extends BaseFragment {
    private AboutUsFragmentBinding binding;
```
I tried to match the default android formatting used by android studio or IntelliJ

Reply:
OK